Sources/SPMBuildCore/PluginInvocation.swift

```diff
@@ -471,6 +473,7 @@ extension PackageGraph {
                     accessibleTools: accessibleTools,
                     writableDirectories: writableDirectories,
                     readOnlyDirectories: readOnlyDirectories,
+                    allowNetworkConnections: [],
```

Comment:
not sure I follow why this is empty in this call?

Reply:
This is the invocation for build tools which do not support specifying permissions.